devicetypes/drzwave/ezmultipli.src/ezmultipli.groovy

```diff
@@ -364,6 +364,7 @@ def ping() {
 
 def configure() {
 	log.debug "OnTime=${settings.OnTime} OnLevel=${settings.OnLevel} TempAdj=${settings.TempAdj}"
+	sendEvent(name: "motion", value: "inactive")
```

Comment:
I recommend putting this in `installed` and setting `displayed: false` so it doesn't show up in the event feed

Reply:
@tpmanley done :)